compiler/src/dotty/tools/dotc/typer/Typer.scala

```diff
@@ -1529,6 +1529,7 @@ class Typer extends Namer
         ctx.error(i"$psym is extended twice", tree.pos)
       seenParents += psym
       if (tree.isType) {
+        checkSimpleKinded(result) // Not needed for constructor calls, as type arguments will be inferred.
```

Comment:
will they be inferred? I'm unconvinced that

```scala
class Siz[T](i: Int)
class Zle extends Siz(1)
```

and 

```scala
class Siz[T]
class Zle extends Siz
```

should be treated differently w/r/t superclass targs.